### PR TITLE
Documentation for #13110

### PR DESCRIPTION
**Pull request by quval**
@katre

### CODE REVIEW

File: site/docs/exec-groups.md [hunk 1]
```diff
@@ -162,3 +170,36 @@ my_rule = rule(
 #
 ```
 
+### Execution groups and platform execution properties
+
+It is possible to define `exec_properties` for arbitrary execution groups on
+platform targets (unlike `exec_properties` set directly on a target, where
+properties for unknown execution groups are rejected). Targets then inherit the
+platform's `exec_properties` that affect the default execution group and any
```
Comment (katre) [inline site/docs/exec-groups.md:178]: "Targets then inherit the execution platform's `exec_proeprties`"

**Review by katre — CHANGES_REQUESTED** [review]
Thanks! I left one comment, but in general can you please clarify that this is coming from execution platforms, not target platforms?

File: site/docs/exec-groups.md (hunk 1 above)
Comment (quval) [reply site/docs/exec-groups.md]: Done (here and in one other place) - not sure if it should be made clearer.

**Review by katre — APPROVED** [review]
I'll merge this today.

**Comment by quval** [comment]
Thank you!

**Pull request closed without merging**